### PR TITLE
Bugfix/245 boolean helper

### LINKED ISSUE #245: Fix manage-UI selection
#241 has broken the manage-UI.
There are a bunch of sentry issues about it. (TypeError: [115](https://sentry.pegnu.cloud/pg/outstanding-cobra/issues/115/), [109](https://sentry.pegnu.cloud/pg/outstanding-cobra/issues/109/), [111](https://sentry.pegnu.cloud/pg/outstanding-cobra/issues/111/))
They need to be fixed.

### PR DESCRIPTION
**Pull request by needToRoll**
This closes #245 

Summary of changes:
- ​fixed said bug (by adding undefind and null checks)
- updated contributing.md to request tests on pr

I am opening this pr because the helper-functions are used all over our frontend

I made sure to:
- [x] Add and / or update tests
- [x] Run the code formatter (`CTRL + ALT + L`)
- [x] Update any documentation (javadoc, domain model, erd, ...)
- [x] Follow the contribution guidelines
- [x] Update CI script and configs on Jenkins (`Jenkinsfile`, `auth0.yml`, `application.yml`, etc.)

Please review @M4R1KU  / cc: @outcobra/developers.

### CODE REVIEW

File: frontend/src/app/core/util/helper.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
+import {isUndefined} from 'util';
+
 export function isFalsy(toCheck: any): boolean {
-    return toCheck == false;
+    return toCheck == false || isNull(toCheck) || isUndefined(toCheck);
```
Comment (jmesserli) [inline frontend/src/app/core/util/helper.ts]: is null or undefined is already contained in the previous check

Comment (jmesserli) [inline frontend/src/app/core/util/helper.ts]:
```typescript
return if (!x) true; else false;
```

File: frontend/src/app/core/util/helper.ts [hunk 2]
```diff
@@ -1,9 +1,13 @@
-export function isFalsy(toCheck: any): boolean {
-    return toCheck == false;
+export function isTruthy(toCheck: any): boolean {
+    if (!toCheck) {
```
Comment (jmesserli) [inline frontend/src/app/core/util/helper.ts]: This has to be `if (toCheck) {`

File: frontend/src/app/core/util/helper.ts [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+        return true;
+    } else {
```
Comment (M4R1KU) [inline frontend/src/app/core/util/helper.ts]: Serious?

Comment (needToRoll) [inline frontend/src/app/core/util/helper.ts]: Yes @jmesserli asked for this implementation

**Comment by M4R1KU** [comment]
We can simplify `isTruthy` more to either `return toCheck ? true : false` or `return Boolean(toCheck)`

**Comment by M4R1KU** [comment]
http://jsbin.com/qefobazeni/edit?js,console

**Pull request merged**